### PR TITLE
[exporter/azuremonitor] Fix: Correct Placement of Span Attributes in AzureMonitorExporter

### PR DESCRIPTION
**Pull request by rajkumar-rangaraj**
**Description:** <Describe what has changed.>

This pull request addresses an issue in the Azure Monitor exporter where span attributes with double and int values were incorrectly added to the `measurements` field instead of the `properties` field in the Application Insights schema.

**Changes**

- Modified the AzureMonitorExporter to ensure that span attributes of type double and int are correctly placed in the properties field.
- Manual testing was conducted to verify that the span attributes appear correctly in the properties field.

**Link to tracking Issue:** <Issue number if applicable> #29683 

**Testing:** <Describe what testing was performed and which tests were added.>

- Updated relevant tests to reflect this change and ensure correctness.

### CODE REVIEW

File: .chloggen/azuremonitorexporter-fix-span-attributes-in-properties.yaml
Comment (pcwiese) [inline .chloggen/azuremonitorexporter-fix-span-attributes-in-properties.yaml:4]: This is now a breaking change I guess?

Comment (rajkumar-rangaraj) [reply .chloggen/azuremonitorexporter-fix-span-attributes-in-properties.yaml]: Changed the bug type to braking - https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/29684/commits/f58089d678ea8df3a20e06150ee9eb2d7aaa355d

**Review by pcwiese — APPROVED** [review]

**Comment by rajkumar-rangaraj** [comment]
@open-telemetry/collector-contrib-approvers Could you please help merge this PR, got an approval from the code owner.

**Review by fatsheep9146 — APPROVED** [review]

**Pull request merged**
